lib/server.js

```diff
@@ -305,23 +323,50 @@ CoAPServer.prototype._handle = function(packet, rsinfo) {
   request.rsinfo = rsinfo
   response.statusCode = '2.05'
   response._request = request._packet
+  response._cachekey = toCacheKey(rsinfo.address, rsinfo.port, packet)
 
   var block2cache = this._block2Cache;
+  var block2cache_expiry = this._options.block2CacheExpiryMs;
```

Comment:
it doesn't appear we did anything w/r/t code style, but for the sake of consistency, can you please zap this semicolon and the one above?